src/output/plugins/PipeWireOutputPlugin.cxx

```diff
@@ -311,6 +311,13 @@ PipeWireOutput::PipeWireOutput(const ConfigBlock &block)
 void
 PipeWireOutput::SetVolume(float _volume)
 {
+	if (thread_loop == nullptr) {
```

Comment:
As I already wrote, this is a bad workaround. Note that this code change doesn't match your commit message: `thread_loop` is **not** set when the output is opened, but when it is enabled.